src/main/java/frc/robot/commands/CmdDriveToPosition.java

```diff
@@ -75,6 +78,8 @@ public void execute()
     public void end(boolean interrupted) 
     {
         _drive.chassisDrive(0, 0, 0);
+
+        RobotLog.getInstance().log("Wanted to go to " + _targetPosition + " with heading " + _targetHeading + " currently at " +  Drive.getInstance().getFieldPosition() + " with heading " + Drive.getInstance().getHeading());
```

Comment:
Same sort of thing here:

```suggestion
        RobotLog.getInstance().log(String.format("Drive to position finished: Current position and heading: %s, %6.2f ; Target position and heading: %s, %6.2f", _drive.getFieldPosition().toString(), _drive.getHeading(), _targetPosition.toString(), _targetHeading));
```